networkit/cpp/centrality/test/CentralityGTest.cpp

```diff
@@ -175,6 +175,16 @@ TEST_F(CentralityGTest, runApproxBetweennessSmallGraph) {
     DEBUG("scores: ", bc);
 }
 
+TEST_F(CentralityGTest, runApproxBetweenness) {
+    DorogovtsevMendesGenerator generator(100);
+    Graph G1 = generator.generate();
+    Graph G(G1, true, false);
+    ApproxBetweenness bc(G, 0.1, 0.1);
+    bc.run();
+    ApproxBetweenness bc1(G1, 0.1, 0.1);
+    bc1.run();
```

Comment:
Should the results from the weighted and the unweighted also be compared to each other?

Reply:
Good point, here I just moved the test to another file, my changes only involve tests for dynamic betweenness. This could be added in a separate PR.